guides/release/routing/specifying-a-routes-model.md

```diff
@@ -86,33 +86,33 @@ method.
 
 Note: A route with a dynamic segment will always have its `model` hook called when it is entered via the URL.
 If the route is entered through a transition (e.g. when using the [link-to](../../templates/links/) Handlebars helper),
-and a model context is provided (second argument to `link-to`), then the hook is not executed.
+and a model context is provided (by passing in `@model` to `<LinkTo />`), then the hook is not executed.
```

Comment:
1. model context -> model object 
2. I think we can just delete (by passing...)